package.json

```diff
@@ -45,8 +40,14 @@
   ],
   "license": "MIT",
   "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "^8.7.0",
+    "eslint-plugin-react": "^7.28.0",
+    "tape": "^5.0.1"
+  },
   "peerDependencies": {
-    "eslint": "^7.12.1",
+    "eslint": "^7.12.1 || ^8.0.0",
```

Comment:
Should be synced with https://github.com/standard/eslint-config-standard/blob/master/package.json